src/GraphQl/Type/SchemaBuilder.php

```diff
@@ -72,6 +72,10 @@ public function getSchema(): Schema
                     continue;
                 }
 
+                if ($resourceMetadata->isInterface()) {
+                    continue;
+                }
+
```

Comment:
I am not sure skipping the operations for interfaces is the correct approach. And if we do, we should at least trigger warnings or something if they are defined anyway.

Reply:
Weak place, agree.
I did this because in my project I dont need mutations in interfaces. And I use custom operations very few.
Additionally I think I had some issues with mutations in interface types, but dont remember details now :(

We can safely move this conditions to include custom query operations here https://github.com/api-platform/core/blob/a1570c836e13d42c9a065f49847e2887410c3c16/src/GraphQl/Type/SchemaBuilder.php#L90